src/components/BookCard.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+/** @component */
+export default BookCard
+
+BookCard.propTypes = {
+  coverUrl: PropTypes.string,
+  title: PropTypes.string.isRequired,
+  author: PropTypes.string.isRequired,
+  description: PropTypes.string,
+}
+
+function BookCard({ coverUrl, title, author, description, id }) {
+  return (
+    <BookCardWrapper>
+      <img src={coverUrl} alt="" />
+      <div>
+        <h3>{title}</h3>
+        <span>
+          von <h4>{author}</h4>
+        </span>
+        <p>{description}</p>
+      </div>
+    </BookCardWrapper>
+  )
+}
+
+const BookCardWrapper = styled.div`
+  position: relative;
+  min-width: 0;
+  display: grid;
+  grid-template-columns: 40% 57%;
```

Comment:
Weird numbers. Please add a comment in the code that explains this. Shouldn't this add up to 100?